content/id/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Mengatur sebuah Pod Untuk menggunakan PersistentVolume untuk Penyimpanan
+content_template: templates/task
+weight: 60
+---
+
+{{% capture overview %}}
+
+Halaman ini akan menjelaskan bagaimana kamu mengatur sebuah Pod untuk menggunakan 
+{{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}}
+untuk penyimpanan.
+Berikut ringkasan prosesnya:
+
+1. Kamu, sebagai seorang administrator klaster, membuat sebuah PersistentVolume yang didukung oleh penyimanan
+fisik. Kamu tidak mengaitkan volume dengan Pod apapun.
+
+2. Kamu, sekarang mengambil peran sebagai seorang _developer_ / pengguna klaster, membuat sebuah
+PersistentVolumeClaim yang secara otomatis terkikat dengan PersistentVolume yang sesuai.
+
+3. Kamu membuat sebuah Pod yang menggunakan PersistentVolumeClaim diatas untuk penyimpanan.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* Kamu membutuhkan sebuah Klaster Kubernetes yang hanya memiliki satu Node, dan 
+{{< glossary_tooltip text="kubectl" term_id="kubectl" >}}
+alat baris perintah yang sudah diatur untuk berkomunikasi dengan klaster kamu. Jika kamu
+tidak memiliki sebuah klaster Node tunggal, kamu dapat membuatnya dengan 
+[Minikube](/docs/getting-started-guides/minikube).
+
+* Familiar dengan materi di
+[Persistent Volumes](/docs/concepts/storage/persistent-volumes/).
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Membuat sebuah berkas index.html di dalam Node kamu
+
+Buka sebuah _shell_ ke Node tunggal di klaster kamu. Bagaiaman kamu membuka sebuah _shell_ tergantung
+dengan bagaiamana kamu mengatur klaster kamu. Contoh, jika kamu menggunakan Minikube, kamu
+dapat membuka sebuah _shell_ ke Node kamu dengan memasukan `minikube ssh`. 
+
+Di dalam _shell_ kamu pada Node itu, buat sebuah direktori dengan nama `/mnt/data`:
+
+```shell
+# Asumsikan Node kamu menggunakan "sudo" untuk menjalankan perintah
+# Sebagai superuser
+sudo mkdir /mnt/data
+```
+
+
+Di dalam  direktori `/mnt/data`, buat sebuah berkas dengan nama `index.html`:
+
+```shell
+# Disini kembali asumsikan bahwa Node kamu menggunakan "sudo" untuk menjalankan perintah
+# Sebagai superuser
+sudo sh -c "echo 'Hello from Kubernetes storage' > /mnt/data/index.html"
+```
+
+{{< note >}}
+Jika Node kamu menggunakan alat untuk akses _superuser_ selain dengan `sudo`, kamu dapat
+membuat ini bekerja jika mengganti `sudo` dengan nama dari alat lainya.
+{{< /note >}}
+
+Menguji bahwa berkas `index.html` ada:
+
+```shell
+cat /mnt/data/index.html
+```
+
+Keluaran akan seperti ini:
+```
+Hello from Kubernetes storage
+```
+
+Sekarang kamu dapat menutup _shell_ di Node kamu.
+
+## Membuat sebuah PersistentVolume
+
+Di latihan, kamu membuat sebuah *hostPath* PersistentVolume. Kubernetes mendukung
+hostPath untuk pengembangan dan pengujian di dalam klaster Node tunggal. Sebuah hostPath
+PersistentVolume menggunakan berkas atau direktori di dalam Node untuk meniru _network-attached storage_.
```

Comment:
NAS is there in RSNI
![image](https://user-images.githubusercontent.com/3192347/83755675-1e312b00-a698-11ea-8431-cd8c6a7142b5.png)

```suggestion
PersistentVolume menggunakan berkas atau direktori di dalam Node untuk meniru penyimpanan terhubung jaringan (NAS, _network-attached storage_).
```